[2025-04-23T18:30:59.717+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T18:30:59.762+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [queued]>
[2025-04-23T18:30:59.789+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [queued]>
[2025-04-23T18:30:59.815+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-23T18:30:59.872+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): ingestao_azure-2446> on 2025-04-22 17:00:00+00:00
[2025-04-23T18:30:59.883+0000] {standard_task_runner.py:72} INFO - Started process 2540 to run task
[2025-04-23T18:30:59.896+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ingestao-blobstorage-azure', 'ingestao_azure-2446', 'scheduled__2025-04-22T17:00:00+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/ingestao-blobstorage-azure.py', '--cfg-path', '/tmp/tmpbpzv6ymf']
[2025-04-23T18:30:59.903+0000] {standard_task_runner.py:105} INFO - Job 81: Subtask ingestao_azure-2446
[2025-04-23T18:31:00.063+0000] {task_command.py:467} INFO - Running <TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [running]> on host e50052774237
[2025-04-23T18:31:00.275+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingestao-blobstorage-azure' AIRFLOW_CTX_TASK_ID='ingestao_azure-2446' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-22T17:00:00+00:00'
[2025-04-23T18:31:00.278+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-23T18:33:43.969+0000] {logging_mixin.py:190} INFO - Erro ao enviar o arquivo 'LD-2446-PE-CAR-PLN-GRC-001-R0.xlsx': [Errno 112] Host is down: '/opt/***/dados/01. LD e CR/2446 - BRASFELS/01. LISTA DE DOCUMENTOS/LD-2446-PE-CAR-PLN-GER-001-R0.xlsx'
[2025-04-23T18:33:43.977+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 45, in upload_files_brasfels
    upload_to_blob(
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 25, in upload_to_blob
    raise e
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 18, in upload_to_blob
    with open(local_file_path, "rb") as data:
OSError: [Errno 112] Host is down: '/opt/airflow/dados/01. LD e CR/2446 - BRASFELS/01. LISTA DE DOCUMENTOS/LD-2446-PE-CAR-PLN-GER-001-R0.xlsx'
[2025-04-23T18:33:44.005+0000] {taskinstance.py:1226} INFO - Marking task as UP_FOR_RETRY. dag_id=ingestao-blobstorage-azure, task_id=ingestao_azure-2446, run_id=scheduled__2025-04-22T17:00:00+00:00, execution_date=20250422T170000, start_date=20250423T183059, end_date=20250423T183344
[2025-04-23T18:33:44.034+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-23T18:33:44.036+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 81 for task ingestao_azure-2446 ([Errno 112] Host is down: '/opt/airflow/dados/01. LD e CR/2446 - BRASFELS/01. LISTA DE DOCUMENTOS/LD-2446-PE-CAR-PLN-GER-001-R0.xlsx'; 2540)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 45, in upload_files_brasfels
    upload_to_blob(
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 25, in upload_to_blob
    raise e
  File "/opt/airflow/dags/ingestao-blobstorage-azure.py", line 18, in upload_to_blob
    with open(local_file_path, "rb") as data:
OSError: [Errno 112] Host is down: '/opt/airflow/dados/01. LD e CR/2446 - BRASFELS/01. LISTA DE DOCUMENTOS/LD-2446-PE-CAR-PLN-GER-001-R0.xlsx'
[2025-04-23T18:33:44.079+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-23T18:33:44.105+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T18:33:44.109+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-23T19:54:58.617+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T19:54:58.641+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [queued]>
[2025-04-23T19:54:58.655+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [queued]>
[2025-04-23T19:54:58.657+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-23T19:54:58.676+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): ingestao_azure-2446> on 2025-04-22 17:00:00+00:00
[2025-04-23T19:54:58.683+0000] {standard_task_runner.py:72} INFO - Started process 3655 to run task
[2025-04-23T19:54:58.688+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ingestao-blobstorage-azure', 'ingestao_azure-2446', 'scheduled__2025-04-22T17:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/ingestao-blobstorage-azure.py', '--cfg-path', '/tmp/tmpv1whev1u']
[2025-04-23T19:54:58.691+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask ingestao_azure-2446
[2025-04-23T19:54:58.738+0000] {task_command.py:467} INFO - Running <TaskInstance: ingestao-blobstorage-azure.ingestao_azure-2446 scheduled__2025-04-22T17:00:00+00:00 [running]> on host e50052774237
[2025-04-23T19:54:58.806+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingestao-blobstorage-azure' AIRFLOW_CTX_TASK_ID='ingestao_azure-2446' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-22T17:00:00+00:00'
[2025-04-23T19:54:58.809+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-23T19:54:59.917+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/brasfels-2446/LD-2446-PE-CAR-PLN-GRC-001-R0.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '4187310'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'd6bbf94e-207c-11f0-9d10-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-23T19:55:13.897+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Wed, 23 Apr 2025 19:55:13 GMT'
    'ETag': '"0x8DD82A0C334EEC3"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': 'f2f4a712-601e-0063-7489-b48f30000000'
    'x-ms-client-request-id': 'd6bbf94e-207c-11f0-9d10-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Wed, 23 Apr 2025 19:55:13 GMT'
[2025-04-23T19:55:13.900+0000] {logging_mixin.py:190} INFO - Arquivo 'LD-2446-PE-CAR-PLN-GRC-001-R0.xlsx' enviado com sucesso para o container 'brasfels-2446'.
[2025-04-23T19:55:14.117+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/brasfels-2446/CS-2446-PE-CAR-PLN-GRC-001-R0.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '80279'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'df32ec68-207c-11f0-9d10-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-23T19:55:15.988+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Wed, 23 Apr 2025 19:55:15 GMT'
    'ETag': '"0x8DD82A0C4702A9B"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '7dda23a2-701e-0040-3789-b415f3000000'
    'x-ms-client-request-id': 'df32ec68-207c-11f0-9d10-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Wed, 23 Apr 2025 19:55:15 GMT'
[2025-04-23T19:55:15.990+0000] {logging_mixin.py:190} INFO - Arquivo 'CS-2446-PE-CAR-PLN-GRC-001-R0.xlsx' enviado com sucesso para o container 'brasfels-2446'.
[2025-04-23T19:55:16.544+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/brasfels-2446/faturamento_2446.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '8935'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'e0a530d8-207c-11f0-9d10-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-23T19:55:17.472+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Wed, 23 Apr 2025 19:55:17 GMT'
    'ETag': '"0x8DD82A0C552FB9A"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '26e3d56d-a01e-006c-1089-b4f95c000000'
    'x-ms-client-request-id': 'e0a530d8-207c-11f0-9d10-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Wed, 23 Apr 2025 19:55:17 GMT'
[2025-04-23T19:55:17.474+0000] {logging_mixin.py:190} INFO - Arquivo 'faturamento_2446.xlsx' enviado com sucesso para o container 'brasfels-2446'.
[2025-04-23T19:55:17.935+0000] {_universal.py:506} INFO - Request URL: 'https://rpeottaprojeto.blob.core.windows.net/brasfels-2446/PAR_2446_BRASFELS_R0.xlsx'
Request method: 'PUT'
Request headers:
    'Content-Length': '120741'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.10.16 (Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'e179769a-207c-11f0-9d10-9e5d745be2f1'
    'Authorization': 'REDACTED'
A body is sent with the request
[2025-04-23T19:55:19.477+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Wed, 23 Apr 2025 19:55:19 GMT'
    'ETag': '"0x8DD82A0C6884AAB"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': '8fbebe06-d01e-003b-1489-b4576f000000'
    'x-ms-client-request-id': 'e179769a-207c-11f0-9d10-9e5d745be2f1'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Wed, 23 Apr 2025 19:55:18 GMT'
[2025-04-23T19:55:19.481+0000] {logging_mixin.py:190} INFO - Arquivo 'PAR_2446_BRASFELS_R0.xlsx' enviado com sucesso para o container 'brasfels-2446'.
[2025-04-23T19:55:19.484+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-23T19:55:19.509+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-23T19:55:19.511+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ingestao-blobstorage-azure, task_id=ingestao_azure-2446, run_id=scheduled__2025-04-22T17:00:00+00:00, execution_date=20250422T170000, start_date=20250423T195458, end_date=20250423T195519
[2025-04-23T19:55:19.558+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-23T19:55:19.575+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T19:55:19.580+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
